src/utils/sortObjectKeys.ts

```diff
@@ -1,22 +1,27 @@
-import { Settings } from "../types/settings";
+import { Settings, SortOrder } from "../types/settings";
+
+function sortObjectKeys(obj: any, sortOrder: SortOrder): any {
+  if (sortOrder === SortOrder.NONE) return obj;
 
-function sortObjectKeys(
-  obj: any,
-  sortOrder: Settings["sort"]
-): any {
-  if (sortOrder === "none") return obj;
   if (Array.isArray(obj)) {
-    return obj.map(item => sortObjectKeys(item, sortOrder));
+    return obj.map((item) => sortObjectKeys(item, sortOrder));
   } else if (obj !== null && typeof obj === "object") {
-    const keys = Object.keys(obj);
-    const sortedKeys =
-      sortOrder === "A-Z" ? keys.sort() : keys.sort().reverse();
-    const sortedObj: any = {};
-    for (const key of sortedKeys) {
-      sortedObj[key] = sortObjectKeys(obj[key], sortOrder);
-    }
-    return sortedObj;
+    // Convertimos el objeto a un array de [clave, valor] y aplicamos recursión en los valores
+    const entries = Object.entries(obj).map(([key, value]) => [
+      key,
+      sortObjectKeys(value, sortOrder),
+    ]);
+
+    // Ordenamos el array según la clave, de forma ascendente o descendente
+    const sortedEntries =
+      sortOrder === SortOrder.ASC
+        ? entries.sort((a, b) => a[0].localeCompare(b[0]))
+        : entries.sort((a, b) => b[0].localeCompare(a[0]));
```

Comment:
Check this comment about abstracting logic in functions (we can call them handlers), this way is less verbose this function and it's more extensible in case in future exists more sort types
https://github.com/heliomar-pena/i18n-populator/pull/88#discussion_r2010210216